org-code-javabuilder/neighborhood/src/main/java/org/code/neighborhood/NeighborhoodSignalKey.java

```diff
@@ -1,6 +1,18 @@
 package org.code.neighborhood;
 
 public enum NeighborhoodSignalKey {
+  // Initialize a new painter
+  INITIALIZE,
   // Move the painter one space forward
-  MOVE
+  MOVE,
+  // Paint the current location
+  PAINT,
+  // Remove all paint from current location
+  REMOVE_PAINT,
+  // Take paint from the bucket
+  TAKE_PAINT,
+  // Hide the painter on the screen
+  HIDE_PAINTER,
+  // Show the painter on the screen
+  SHOW_PAINTER
```

Comment:
One item that wasn't on the list but we'll also need to add is HIDE_BUCKETS to make all the buckets on a level disappear.

Reply:
Definitely can be a follow-up. Looks like this was missed even on the API description.

Reply:
it's a quick update, I'll add it